guides/common/modules/con_overview-of-usage-metrics-collection-in-project.adoc

```diff
@@ -3,12 +3,20 @@
 
 {Project} includes the `satellite-maintain report generate` command, which can collect information from user environments to better understand how {Project} is used.
 
-A `systemd` timer named `satellite-usage-metrics-generate.timer` runs weekly to collect the data.
-The timer executes the `{foreman-maintain} report generate` command on your {ProjectServer} and then stores the collected data in the `/var/lib/foreman-maintain/satellite_metrics.yml` file.
+The following `systemd` timers collect the data:
 
-When you use the `sos report` command to generate a report for Red{nbsp}Hat Technical Support, the `satellite_metrics.yml` file is included in the report along with the other configuration, diagnostics, and troubleshooting data that `sos` collects.
+`satellite-usage-metrics-generate.timer`::
+This timer collects the full list of usage metrics.
+It executes the `{foreman-maintain} report generate` command on your {ProjectServer} and then stores the collected data in the `/var/lib/foreman-maintain/satellite_metrics.yml` file.
+The timer runs weekly.
+
+`satellite-usage-metrics-condense.timer`::
+This timer collects the same set of metrics as `satellite-usage-metrics-generate.timer`, condenses them so that they can be uploaded as custom Red{nbsp}Hat Subscription Manager facts, and stores them in the `/etc/rhsm/facts/foreman.yml` file.
```

Comment:
```suggestion
This timer collects the same set of metrics as `satellite-usage-metrics-generate.timer`, condenses them so that they can be uploaded as custom Red{nbsp}Hat Subscription Manager facts, and stores them in the `/etc/rhsm/facts/foreman.facts` file.
```